plugins/org.jkiss.dbeaver.ext.altibase/src/org/jkiss/dbeaver/ext/altibase/AltibaseConstants.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2023 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.ext.altibase;
+
+import org.jkiss.utils.StandardConstants;
+
+public class AltibaseConstants {
+    
+    public static final String NEW_LINE = System.getProperty(StandardConstants.ENV_LINE_SEPARATOR);
+    public static final String SPACE4 = "    ";
+    public static final String PSM_POSTFIX = ";" + AltibaseConstants.NEW_LINE + "/";
```

Comment:
Not used variable.

Reply:
Okay SPACE4, PSM_POSTFIX will be removed. NEW_LINE is used a lot.